client/src/partials/patients/record/patient_record.html

```diff
@@ -0,0 +1,167 @@
+                  </div>
+
+                  <div class="form-group">
+                    <label class="control-label">{{ "TABLE.COLUMNS.GENDER" | translate }}</label>
+                    <p class="form-control-static" id="gender">{{::PatientRecordCtrl.patient.sex}}</p>
+                  </div>
+
+                </div>
+              </div>
+
+              <div class="col-sm-4">
+                <div style="border : 1px solid #ccc; height : 200px;">
+                  <!-- no image placeholder -->
+                  <div ng-if="!PatientRecordCtrl.patient.image" style="text-align : center;">
+                    <span style="font-size : 3em; color : #ccc; padding-top: 70px;" class="glyphicon glyphicon-picture"></span>
+                  </div>
+                </div>
+
+                <a class="btn btn-default btn-block" disabled><span class="glyphicon glyphicon-upload"></span> {{ "PATIENT_RECORDS.UPLOAD_PICTURE" | translate }}</a>
+                <a
+                  data-method="edit"
+                  ng-href="/#/patients/{{::PatientRecordCtrl.patient.uuid}}/edit"
+                  class="btn btn-default btn-block">
+                  <span class="glyphicon glyphicon-edit"></span>
+                  {{ "PATIENT_EDIT.EDIT_DETAILS" | translate }}
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div> <!-- End patient details row -->
+
+      <div class="row">
+
+        <!-- Patient Activity Panel -->
+        <div class="col-sm-6">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <span class="glyphicon glyphicon-time"></span> Activity
+            </div>
+            <div class="panel-body" style="height : 250px;">
+
+              <!-- Placeholder - not implemented -->
+              <div style="text-align : center; font-size : 3em; color : #ccc; padding-top : 80px;">
+                <span class="glyphicon glyphicon-wrench"></span> Not Implemented
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <!-- Patient Bill Panel -->
+        <div class="col-sm-6">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <span class="glyphicon glyphicon-file"></span> Patient Visit
+            </div>
+            <div class="panel-body" style="height : 250px;">
+
+              <!-- Placeholder - not implemented -->
+              <div style="text-align : center; font-size : 3em; color : #ccc; padding-top : 80px;">
+                <span class="glyphicon glyphicon-wrench"></span> Not Implemented
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="row">
+        <!-- Patient Documents -->
+        <div class="col-sm-6">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <span class="glyphicon glyphicon-paperclip"></span> Patient Documents
```

Comment:
These are all just placeholder elements - translations should be included when they are implemented. I guess TODO implement this.